pkgs/development/coq-modules/coq-bits/default.nix

```diff
@@ -22,7 +22,7 @@ with lib; mkCoqDerivation {
     };
   };
 
-  propagatedBuildInputs = [ mathcomp.algebra ];
+  propagatedBuildInputs = [ mathcomp.algebra mathcomp.ssreflect mathcomp.fingroup ];
```

Comment:
Is this change related to the purpose of this PR?

Reply:
Yes, I can't build the package without it. But I have now moved them to `extraBuildInputs` instead